docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md

```diff
@@ -0,0 +1,89 @@
+---
+id: jamf-source
+title: Jamf Source
+sidebar_label: Jamf
+tags:
+  - cloud-to-cloud
+  - jamf
+description: Learn how to collect data from the Jamf platform.
+---
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/jamf/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/jamf/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/jamf/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/jamf.png')} alt="thumbnail icon" width="85"/>
+
+Jamf is a software company that specializes in mobile device management (MDM) and endpoint management solutions for Apple devices such as Macs, iPhones, iPads, and Apple TVs. This Jamf integration helps you to ingest inventory data using the [Jamf Pro API](https://developer.jamf.com/jamf-pro/v11.4.0/docs/jamf-pro-api-overview) and [Jamf Classic API](https://developer.jamf.com/jamf-pro/v11.4.0/docs/getting-started-2).
+
+## Data collected
+
+| Polling Interval | Data |
+| :--- | :--- |
+| 24 hours |  [Inventory ID](https://developer.jamf.com/jamf-pro/reference/get_v1-computers-inventory) <ul><li>[Computer History](https://developer.jamf.com/jamf-pro/reference/findcomputerhistorybyid)</li><li>[Computer Management](https://developer.jamf.com/jamf-pro/reference/findcomputermanagementbyid)</li></ul> |
+
+
+## Setup
+
+### Vendor configuration
+
+The Jamf source requires you to provide the Base URL, Client ID, and Client Secret to access the source data.
+
+- The Base URL is used to retrieve the source data from the Jamf. The **Base URL** for Jamf is `https://yourServer.jamfcloud.com`.
```

Comment:
Could you please add below line after the URL?
`Replace "yourServer" with your organization server.`